test/e2e/tests/surveySchemaSummary.js

```diff
@@ -0,0 +1,96 @@
+import fs from 'fs'
+import path from 'path'
+import csv from 'csv/lib/sync'
+
+import { DataTestId, getSelector } from '../../../webapp/utils/dataTestId'
+import { downloadsPath } from '../paths'
+import { gotoFormDesigner } from './_navigation'
+import { cluster } from '../mock/nodeDefs'
+
+const getTestNodeDefsOrderedByPath = () => {
+  const items = []
+
+  const stack = [{ parentPath: null, nodeDef: cluster }]
+  while (stack.length) {
+    const nodeDefItem = stack.shift()
+    const { nodeDef, parentPath } = nodeDefItem
+    const nodeDefPath = parentPath ? `${parentPath}.${nodeDef.name}` : nodeDef.name
+
+    items.push({ nodeDef, path: nodeDefPath })
+
+    if ('children' in nodeDef) {
+      stack.push(
+        ...Object.values(nodeDef.children).map((nodeDefChild) => ({
+          parentPath: nodeDefPath,
+          nodeDef: nodeDefChild,
+        }))
+      )
+    }
+  }
+  // sort items by path
+  return items.sort((item1, item2) => {
+    const path1 = item1.path
+    const path2 = item2.path
+    if (path1 > path2) return 1
+    if (path2 > path1) return -1
+    return 0
+  })
+}
```

Comment:
maybe we can extract this because it is repeated

Reply:
but this is a tests! okay!

Reply:
uhm, right, where can we put it? it's something that can go to some ArrayUtils module, something like sortByProperty